plugins/defaultplugins/common/model/nat/nat.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+package nat;
+
+/* NAT44 global config */
+message Nat44Global {
+    uint32 vrfId = 1;                           /* VRF (table) ID */ 
+    bool forwarding = 2;                        /* Enable/disable forwarding */
+    repeated string in = 3;                     /* Local network interfaces enabled for NAT44 */
+    repeated string out = 4;                    /* External network interfaces enabled for NAT44 */
+    message AddressPool {
+        string FirstSrcAddress = 1;             /* starting address of the source IPv4 pool range */
+        string LastSrcAddres = 2;               /* ending address of the source IPv4 pool range. Optional parameter */
+        bool twiceNat = 3;                   
+    }
+    repeated AddressPool address_pool = 5;
+}
```

Comment:
We will also need a pool of "identity mappings" in the global config. Check out https://wiki.fd.io/view/VPP/NAT for "Identity NAT44". 
Binary APIs: nat44_add_del_identity_mapping + nat44_identity_mapping_dump